scripts/etcd-mount/etcd-mc.yaml

```diff
@@ -0,0 +1,192 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 98-var-lib-etcd
+spec:
+  config:
+    ignition:
+      version: 3.1.0
+    storage:
+      files:
+        - path: /etc/find-secondary-device
+          mode: 0755
+          contents:
+            source: data:text/plain;charset=utf-8;base64,IyEvYmluL2Jhc2gKc2V0IC11byBwaXBlZmFpbAoKZm9yIGRldmljZSBpbiAvZGV2L3ZkKjsgZG8gCi91c3Ivc2Jpbi9ibGtpZCAkZGV2aWNlICY+IC9kZXYvbnVsbArCoGlmIFsgJD8gPT0gMiDCoF07IHRoZW4KwqAgwqAgZWNobyAic2Vjb25kYXJ5IGRldmljZSBmb3VuZCAkZGV2aWNlIgrCoCDCoCBlY2hvICJjcmVhdGluZyBmaWxlc3lzdGVtIGZvciBldGNkIG1vdW50IgrCoCDCoCBta2ZzLnhmcyAtTCB2YXItbGliLWV0Y2QgLWYgJGRldmljZSAmPiAvZGV2L251bGwKwqAgwqAgdWRldmFkbSBzZXR0bGUKwqAgwqAgdG91Y2ggL2V0Yy92YXItbGliLWV0Y2QtbW91bnQKwqAgwqAgZXhpdArCoGZpCmRvbmUKZWNobyAiQ291bGRuJ3QgZmluZCBzZWNvbmRhcnkgYmxvY2sgZGV2aWNlISIgPiYyCmV4aXQgNzcK 
```

Comment:
#!/bin/bash
set -uo pipefail

for device in /dev/vd*; do 
/usr/sbin/blkid $device &> /dev/null
 if [ $? == 2  ]; then
    echo "secondary device found $device"
    echo "creating filesystem for etcd mount"
    mkfs.xfs -L var-lib-etcd -f $device &> /dev/null
    udevadm settle
    touch /etc/var-lib-etcd-mount
    exit
 fi
done
echo "Couldn't find secondary block device!" >&2
exit 77

Reply:
Script content - after decoding